roles/znc/tasks/main.yml

```diff
@@ -3,28 +3,30 @@
   file:
     path: "{{ item }}"
     state: directory
+    # mode: 0755
   with_items:
     - "{{ znc_data_directory }}"
 
-- name: ZNC
+- name: Create ZNC Docker Container
   docker_container:
     name: znc
     image: znc
     pull: true
     volumes:
       - "{{ znc_data_directory }}:/znc-data:rw"
       - "/etc/timezone:/etc/timezone:ro"
+    network_mode: "bridge"
```

Comment:
Not clear why this was added?

Reply:
To give confirmation that the container is run on the bridge mode network as apposed to the host network. It also makes it easy for someone to change a containers network mode if they wanted to use a different method, like overlay or macvlan. It's also on all of the other PRs I've moved over to roles and on a few others, such as openhab.

Reply:
OK, I know this is picky but I'm going to remove these as I merge the PRs (which I am very grateful for!). 

Few reasons for that:
- This is a change to what's currently being done at the moment.  
- Adding options that are set to defaults increases the amount of stuff required to be understood. Docker's default is bridge networking, so people shouldn't be surprised by the network mode. 
- If we really think people need to change it, then it should be set as a parameter people can set through role variables

Reply:
I can add that functionality